backend/src/file/file.go

```diff
@@ -0,0 +1,159 @@
+package file
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"strings"
+
+	"github.com/GenerateNU/sac/backend/src/config"
+	"github.com/GenerateNU/sac/backend/src/errors"
+	"github.com/GenerateNU/sac/backend/src/models"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/google/uuid"
+)
+
+type FileTypes string
```

Comment:
```suggestion
type FileType string
```
nit for naming convention sake